<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Board</class>
    <widget class="QDialog" name="Board">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>668</width>
                <height>403</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Task</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>1</y>
                    <width>641</width>
                    <height>401</height>
                </rect>
            </property>
            <layout class="QGridLayout" name="TaskLayout_3">
                <item row="0" column="0">
                    <layout class="QHBoxLayout" name="toolbarLayout">
                        <item>
                            <widget class="QLabel" name="statusLabel">
                                <property name="text">
                                    <string>Status: 0 Checklist</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="horizontalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="addTaskButton">
                                <property name="text">
                                    <string>Add Task</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="SaveTask">
                                <property name="text">
                                    <string>Save</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item row="1" column="0">
                    <widget class="QToolBox" name="toolBox">
                        <widget class="QWidget" name="Page1_2">
                            <property name="geometry">
                                <rect>
                                    <x>0</x>
                                    <y>0</y>
                                    <width>639</width>
                                    <height>335</height>
                                </rect>
                            </property>
                            <attribute name="label">
                                <string/>
                            </attribute>
                            <layout class="QVBoxLayout" name="TaskLayout">
                                <property name="sizeConstraint">
                                    <enum>QLayout::SetMaximumSize</enum>
                                </property>
                            </layout>
                        </widget>
                    </widget>
                </item>
            </layout>
        </widget>
    </widget>
    <layoutdefault spacing="6" margin="11"/>
    <resources/>
    <connections/>
</ui>
